### DISCUSSION TITLE
Where to report issues with GitHub docs

### DISCUSSION THREAD

**Discussion by bvwells** [community/community#22071] [New to GitHub]
<p>I noticed a broken link in GitHub docs and couldn’t find where to report this issue so trying here.</p>
<p>The broken link is in the section here:</p>
<p><a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/personalizing-your-profile#displaying-badges-on-your-profile" class="onebox" target="_blank" rel="nofollow noopener">https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/personalizing-your-profile#displaying-badges-on-your-profile</a></p>
<p>The link:</p>
<p>see “GitHub’s products”</p>
<p>wrongly links to</p>
<p><a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/github/getting-started-with-github/githubs-products#github-pro" class="onebox" target="_blank" rel="nofollow noopener">https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/github/getting-started-with-github/githubs-products#github-pro</a></p>
<p>It would also be nice if there was a report a docs issue link on the docs site.</p>
<p>Hope this helps!<br>
Ben</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi <a class="mention" href="/bvwells">@bvwells</a>! :wave: Welcome to the Community!</p>
<p>It looks like that error has already been caught and fixed, but thank you for reporting it!</p>
<p>We’re always working to improve GitHub, and we consider every suggestion we receive, so perhaps you’d like to submit a feature request <a href="https://support.github.com/contact/feedback">through our official product feedback form</a> so that our product team can see you’d like to be able to report issues with the docs right there on the page?</p>

**Comment by bvwells** [comment]
<p>Thanks for your help <a class="mention" href="/yamiacat">@yamiacat</a>!</p>
